packages/geocoder/src/geocoders/otp.ts

```diff
@@ -21,7 +21,23 @@ export default class OTPGeocoder extends Geocoder {
 
 
   rewriteAutocompleteResponse(response: OTPGeocoderResponse): MultiGeocoderResponse {
-    const generateLabel = stop => stop.code ? `${stop.name} (${stop.code})` : stop.name
+
+    const generateLabel = stop => {
```

Comment:
Does this function need to be inside `rewriteAutocompleteResponse`? It doesn't seem like you're using the closure, so it could probably be outside. Maybe just put it at the top of the file?